From e6d6daf58847fd07ea055d9563b9c220a2fd1122 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1s=20Gonz=C3=A1lez?=
 <tomasagustin.gonzalezorlando@arm.com>
Date: Thu, 14 Dec 2023 11:05:02 +0000
Subject: [PATCH] ci.sh: Update parsec version to 1.3.0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Currently, parsec 0.6.0 is being used to test the parsec-se-driver.

 * Use parsec 1.3.0 to test instead.

Upstream-Status: Submitted [https://github.com/parallaxsecond/parsec-se-driver/pull/36]
Signed-off-by: Tomás González <tomasagustin.gonzalezorlando@arm.com>
---
 ci/ci.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ci/ci.sh b/ci/ci.sh
index 31688eb..b7a5260 100755
--- a/ci/ci.sh
+++ b/ci/ci.sh
@@ -43,7 +43,7 @@ tpm2_startup -T mssim
 mkdir /run/parsec
 
 # Install and run Parsec
-git clone --branch 0.6.0 https://github.com/parallaxsecond/parsec
+git clone --branch 1.3.0 https://github.com/parallaxsecond/parsec
 pushd parsec
 cargo build --features tpm-provider --release
 ./target/release/parsec -c ../ci/config.toml &
-- 
2.25.1

